socorro/collector/wsgi_breakpad_collector.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, config):
         self.accept_submitted_legacy_processing = \
             self._get_accept_submitted_legacy_processing()
         self.throttler = self._get_throttler()
+        self.metrics = self._get_metrics()
```

Comment:
This is how most of the components are structured. I'm not entirely clear on why it's this way. In some cases, the `_get*` methods cache the results. One nice thing about it is that it's easy to mock out in tests, though I didn't mock out `_get_metrics` anywhere.

I think I'm hesitant to change this way of doing things at this juncture.